styles/globals.css

```diff
@@ -40,3 +40,19 @@ input[type="search"]::-webkit-search-cancel-button {
 .z-50 {
   z-index: 50;
 }
+
+.-left-17 {
```

Comment:
remove this please, relative positioning and margins will solve this problem

Reply:
Even using relative positioning and margins, it won't align the dropdown.
IMO, defining a `left` or `margin` property with the value of  `-4.4rem` or using arbitrary values is the way to go.
@0-vortex what do you think about it?